### DISCUSSION TITLE
Welcome to HandTracking Discussions!

### DISCUSSION THREAD

**Discussion by israr96418** [israr96418/HandTracking#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ML-Engineer12** [comment]
A very nice project.
This module is used in alote of project like volume control in hand, virtual painting etc

**Comment by israr96418** [comment]
Yeah , I create this module specially for "volume control by hand and virtual painting"
